packages/ckeditor5-engine/docs/framework/guides/deep-dive/custom-element-conversion.md

```diff
@@ -274,6 +270,75 @@ editor.conversion.for( 'dataDowncast' )
 	.add( dispatcher => dispatcher.on( 'insert:infoBox', dataDowncastConverter ) );
 ```
 
+### The model to view position mapping
+
+After adding the above downcast converters we would end up with the following view structure:
+
+```
```

Comment:
It'd help to show (maybe in a two-column table) what model translates to what view.

Also, what is the `[]` below for?